src/Calculator/Common/KeyboardShortcutManager.h

```diff
@@ -60,20 +55,31 @@ namespace CalculatorApp
 
             static void OnVirtualKeyShiftChordPropertyChanged(Windows::UI::Xaml::DependencyObject ^ target, MyVirtualKey oldValue, MyVirtualKey newValue);
 
-            static void OnVirtualKeyInverseChordPropertyChanged(Windows::UI::Xaml::DependencyObject ^ target, MyVirtualKey oldValue, MyVirtualKey newValue);
-
-            static void
-            OnVirtualKeyControlInverseChordPropertyChanged(Windows::UI::Xaml::DependencyObject ^ target, MyVirtualKey oldValue, MyVirtualKey newValue);
-
             static void OnVirtualKeyAltChordPropertyChanged(Windows::UI::Xaml::DependencyObject ^ target, MyVirtualKey oldValue, MyVirtualKey newValue);
 
             static void
             OnVirtualKeyControlShiftChordPropertyChanged(Windows::UI::Xaml::DependencyObject ^ target, MyVirtualKey oldValue, MyVirtualKey newValue);
 
             static void OnCharacterReceivedHandler(Windows::UI::Core::CoreWindow ^ sender, Windows::UI::Core::CharacterReceivedEventArgs ^ args);
             static void OnKeyDownHandler(Windows::UI::Core::CoreWindow ^ sender, Windows::UI::Core::KeyEventArgs ^ args);
-            static void OnKeyUpHandler(Windows::UI::Core::CoreWindow ^ sender, Windows::UI::Core::KeyEventArgs ^ args);
             static void OnAcceleratorKeyActivated(Windows::UI::Core::CoreDispatcher ^, Windows::UI::Core::AcceleratorKeyEventArgs ^ args);
+            static const std::multimap<MyVirtualKey, Platform::WeakReference>* KeyboardShortcutManager::GetCurrentKeyDictionary(bool altPressed = false);
+
+        private:
+            static std::map<int, std::multimap<wchar_t, Platform::WeakReference>> s_CharacterForButtons;
+            static std::map<int, std::multimap<MyVirtualKey, Platform::WeakReference>> s_VirtualKeysForButtons;
+            static std::map<int, std::multimap<MyVirtualKey, Platform::WeakReference>> s_VirtualKeyControlChordsForButtons;
+            static std::map<int, std::multimap<MyVirtualKey, Platform::WeakReference>> s_VirtualKeyShiftChordsForButtons;
+            static std::map<int, std::multimap<MyVirtualKey, Platform::WeakReference>> s_VirtualKeyAltChordsForButtons;
+            static std::map<int, std::multimap<MyVirtualKey, Platform::WeakReference>> s_VirtualKeyControlShiftChordsForButtons;
+
+            static std::map<int, bool> s_IsDropDownOpen;
+            static std::map<int, bool> s_ignoreNextEscape;
+            static std::map<int, bool> s_keepIgnoringEscape;
+            static std::map<int, bool> s_fHonorShortcuts;
+            static std::map<int, bool> s_fDisableShortcuts;
+
+            static Concurrency::reader_writer_lock s_keyboardShortcutMapLock;
```

Comment:
Looks like this isn't used anymore?

Reply:
`s_keyboardShortcutMapLock`? It's used in many methods in KeyboardShortcutManager.cpp

Reply:
Oops, I was looking at the diff wrong.